lib/library.go

```diff
@@ -110,62 +78,24 @@ func CallRPC(inputJSON *C.char) *C.char {
 // just modified to handle the function arg passing
 //export CreateAccount
 func CreateAccount(password *C.char) *C.char {
-	address, pubKey, mnemonic, err := statusAPI.CreateAccount(C.GoString(password))
-
-	errString := ""
-	if err != nil {
-		fmt.Fprintln(os.Stderr, err)
-		errString = err.Error()
-	}
-
-	out := common.AccountInfo{
-		Address:  address,
-		PubKey:   pubKey,
-		Mnemonic: mnemonic,
-		Error:    errString,
-	}
-	outBytes, _ := json.Marshal(out)
+	accountInfo := statusAPI.CreateAccount(C.GoString(password))
+	outBytes, _ := json.Marshal(accountInfo)
```

Comment:
Please handle error as above:
```go
resp, err := json.Marshal(accountInfo)
if err != nil {
  return makeJSONResponse(err)
}
return C.CString(string(resp))
```

Reply:
I don't think Marshaling accountInfo can ever return an error, I couldn't find any way to create a unit test that would cause it to return an error, which is why I thought the uses of outBytes, _ := json.Marshal(out) was common in the existing code.